install/kubernetes/helm/istio/charts/certmanager/templates/rbac.yaml

```diff
@@ -9,12 +9,7 @@ rules:
     resources: ["certificates", "issuers", "clusterissuers"]
     verbs: ["*"]
   - apiGroups: [""]
-    # TODO: remove endpoints once 0.4 is released. We include it here in case
```

Comment:
I think we cannot delete this, as the current cert manager version 0.3

Reply:
Thanks for clarify this, close it.